serde-api/src/main/java/io/micronaut/serde/config/DefaultSerdeConfiguration.java

```diff
@@ -59,7 +61,8 @@ final class DefaultSerdeConfiguration implements SerdeConfiguration {
                               @Bindable(defaultValue = "io.micronaut") List<String> includedIntrospectionPackages,
                               @Bindable(defaultValue = LimitingStream.DEFAULT_MAXIMUM_DEPTH + "") int maximumNestingDepth,
                               @Bindable(defaultValue = DEFAULT_INET_ADDRESS_AS_NUMERIC + "") boolean inetAddressAsNumeric,
-                              @Nullable String propertyNamingStrategy) {
+                              @Nullable String propertyNamingStrategy,
+                              @Property(name = "jackson.json-view.enabled", defaultValue = "false") boolean jsonViewEnabled) {
```

Comment:
Please add a `micronaut.serde.json-view.enabled` alternative and a note in the documentation.